sljit_src/sljitNativeX86_common.c

```diff
@@ -412,7 +412,7 @@ static SLJIT_INLINE void sljit_unaligned_store_sw(void *addr, sljit_sw value)
 /*    Utility functions                               */
 /******************************************************/
 
-static void execute_cpu_id(sljit_u32 info[4])
+static void execute_cpu_id(const sljit_u32 *info)
```

Comment:
The block is replaced, it cannot be const.

Reply:
only the pointer is const, the data inside is changed and updated by both "cpuid" implementations AFAIK (tested also in Windows with MSVC)

Reply:
My MSVC complained when there was no `(int*)` and `(int)` type casts.

Reply:
not sure what you mean, I removed the `int` casts in the last version, because the local alias should work as an alternative and didn't see any warnings, neither would expect any.